setup.py

```diff
@@ -74,6 +74,7 @@
     'sentry': [
         'raven~=6.0,>=6.2.1',
         'scrapy-sentry',
+        'sentry-sdk==1.3.0',
```

Comment:
also included on L44. Why is this added twice? BTW, do we still need `raven` and `scrapy-sentry` now that you've changed the sentry client?

Reply:
I think you can also remove https://github.com/inspirehep/hepcrawl/blob/7cd64d4887b38127d8a0755423929b32b7da4cfa/hepcrawl/settings.py#L110

Reply:
`scrapy-sentry`s readme file says it doesn't report the errors in scrapy pipelines.

Reply:
but you've added a new extension using `sentry-sdk`, which is precisely why you can remove `scrapy-sentry` and `raven` now.